user_docs/en/userGuide.t2t

```diff
@@ -2093,6 +2100,39 @@ Please check your BrailleNote's documentation to find where these keys are locat
 | Windows key | space+dot2+dot4+dot5+dot6 (space+w) |
 | Alt key | space+dot1+dot3+dot4 (space+m) |
 | Toggle input help | space+dot2+dot3+dot6 (space+lower h) |
+
+Following are commands assigned to BrailleNote QT.
+%kc:beginInclude

```

Comment:
You can ask QT to emulate braille dots by pressing a key (it's a toggle). I can clarify this.